pootle/static/js/stats.js

```diff
@@ -366,11 +370,37 @@ const stats = {
 
       $('#js-stats-checks').show();
     }
-
     this.hasChecksData = true;
     this.toggleChecksVisibility();
-  }
+  },
+
+  checksStateIsOpen() {
+    return (window.location.search === '?details');
+  },
+
+  checksStatePush() {
+    if (window.history && window.history.pushState) {
+      if (!this.isExpanded) {
+	window.history.pushState("", "", this.pootlePath);
+      } else {
+	window.history.pushState("", "", "?details");
+      }
+    }
+  },
 
+  reopenChecks() {
```

Comment:
This is unintuitive. First off, it's taking more responsibilities than opening the details — it's also overwriting the `onpopstate` handler for `window`.

Also, arguably `reopen` has misleading semantics: this is not opening anything *again*.